sql/updates/latest-dev.sql

```diff
@@ -161,4 +161,19 @@ DROP FUNCTION IF EXISTS _timescaledb_functions.get_chunk_colstats;
 DROP FUNCTION IF EXISTS _timescaledb_internal.get_chunk_relstats;
 DROP FUNCTION IF EXISTS _timescaledb_internal.get_chunk_colstats;
 
-
+-- In older TSDB versions, we disabled autovacuum for compressed chunks
+-- to keep the statistics. However, this restriction was removed in
+-- #5118 but no migration was performed to remove the custom
+-- autovacuum setting for existing chunks.
+DO $$
+DECLARE
+  chunk regclass;
+BEGIN
+  FOR chunk IN
+  SELECT pg_catalog.format('%I.%I', schema_name, table_name)::regclass
+    FROM _timescaledb_catalog.chunk
+  LOOP
+    EXECUTE pg_catalog.format('ALTER TABLE %s RESET (autovacuum_enabled);', chunk::text);
+  END LOOP;
+END
+$$;
```

Comment:
Isn't possible the parent table (hypertable) also have the autovacuum disabled?

Reply:
our code would never set it on the hypertable only on individual chunks

Reply:
@fabriziomello This is possible, but this is, as far as I know, then configured by the user. 

The intention of the PR was to reset only the setting we did in our old compression code. Not sure, if we should also reset these flags on the parent table.